Sources/EmojiPickerViewController/Models/Loader/EmojiLoader.swift

```diff
@@ -304,6 +305,13 @@ class EmojiLoader: Loader {
                 let character = Character(String(unicodeScalarView))
 
                 let emoji = Emoji(character: character, status: data.status, cldrOrder: emojiOrder, group: String(group), subgroup: String(subgroup))
+                
+                var isValid: Bool {
```

Comment:
The intention you made here is even if the version of the platform is old and can't handle new emoji, this `guard-else` statement guards the troublesome emoji, right? 🧐